### PR TITLE
ITOP-4624: Create FB activity-composer-vue and update projects versions/dependencies

### PR DESCRIPTION
**Pull request by aelbenney**

### CODE REVIEW

File: webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue [hunk 1]
```diff
@@ -210,6 +231,7 @@ export default {
     'news.summary': function(newValue, oldValue) { if(newValue !== oldValue) { this.autoSave(); } },
     'news.body': function(newValue, oldValue) { if(newValue !== oldValue) { this.autoSave(); } },
     'news.attachments': function() {
+      console.log('atta',this.news.attachments);
```
Comment (thomasdelhomenie) [inline webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue]: To remove.

File: webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue [hunk 2]
```diff
@@ -611,6 +632,13 @@ export default {
       } else {
         window.open('/', '_self');
       }
+    },
+    openApp() {
+      document.getElementsByClassName('attachments drawer')[0].className += ' open';
+      document.getElementById('exoAttachmentsApp').getElementsByClassName('drawer-backdrop')[0].style.display = 'block';
+    },
+    attachFiles(files) {
+      console.log('FROMCOMP',files);
```
Comment (thomasdelhomenie) [inline webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue]: To remove.

File: webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue [hunk 3]
```diff
@@ -611,6 +632,13 @@ export default {
       } else {
         window.open('/', '_self');
       }
+    },
+    openApp() {
+      document.getElementsByClassName('attachments drawer')[0].className += ' open';
```
Comment (thomasdelhomenie) [inline webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue]: This class (`'attachments drawer'`) is inside the exo-attachments component. We should not refer to elements inside the component. The component should be seen as a black box. We should ether not include the drawer inside the component, expose a props in the component to allow to show/hide it.

File: webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue (hunk 1 above)
Comment (aelbenney) [reply webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue]: done

File: webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue (hunk 2 above)
Comment (aelbenney) [reply webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue]: done

**Pull request merged**